apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServiceAutoConfiguration.java

```diff
@@ -73,9 +74,12 @@ public ConfigService configService() {
       return new ConfigServiceWithCache(releaseService, releaseMessageService,
           grayReleaseRulesHolder(), bizConfig, meterRegistry);
     }
+    if(bizConfig.isConfigServiceChangeCacheEnabled()){
```

Comment:
I think the `config-service.cache.enabled` and `config-service.incremental.change.enabled` can be combined.
The first and third scenarios are new.

| config-service.cache.enabled | config-service.incremental.change.enabled |
|------------------------------|-------------------------------------------|
| true                         | true                                      |
| true                         | false                                     |
| false                        | true                                      |
| false                        | false                                     |